io/src/ply_io.cpp

```diff
@@ -306,52 +306,26 @@ namespace pcl
     vertex_offset_before_ += static_cast<int> (sizeof (ContentType));
   }
 
-  template <>
-  std::tuple<std::function<void (pcl::io::ply::uint8)>, std::function<void (pcl::io::ply::int32)>, std::function<void ()> >
+  template <typename SizeType, typename ContentType>
```

Comment:
This is now fully templated. Should it be moved to a header file?

Reply:
I think that is not necessary, because `listPropertyDefinitionCallback` is only used in ply_io.cpp, and the definition does not have to be available anywhere else.

Reply:
So long as the downstream users face no issues, we can keep it as it is 😄